### ISSUE TITLE
[CAIA A11y Research]: Benefits Condition Team: Conditions Information Entry (December 2024)

### ISSUE THREAD

**Issue by sudeeptibh** [department-of-veterans-affairs/va.gov-team#96633]
> [!NOTE]
> **High-Level Overview**: Benefits Condition Team
> - **Research Timeframe**: Week of 12/2/2024 
> - **Pilot**:  `Completed 11/25/2024` Mobile  97582`
> - **Assistive Tech Users**: `11 ` AT Users Recruited
>    - [**Schedule Source of Truth:** Available via Canvas in Slack](https://dsva.slack.com/canvas/C082SKRHAJV) 
> - **Artifacts**:  [Research Plan](https://docs.google.com/document/d/1I4YAFI9xYL6EnMV6DQ5iqdEUwgCKR_TWXich9JvW-Zo/edit?tab=t.0), [Convo Guide](https://docs.google.com/document/d/12bLzBBviYDFCwGKvPUU8CmESkpqbV3MrDcyik-WumeI/edit?tab=t.0#heading=h.mtc4swvlio8d)
> - **Testing Environment:** `Staging `
> - **Assistive Tech Being Used**: JAWS, VoiceOver, TalkBack, Screen Magnification/ZoomText, Alternative Navigation - voice control and  Alternative navigation - one-handed keyboard
> - **Task Lists**: See task lists for next steps/definition of done

## CAIA Assignee(s)

<details><summary>Toggle to view information about the CAIA Team.</summary>

- @mutiaadhocteam 
- @SarahKay8 
- @sara-amanda
- @EvanAtCoforma 
- @coforma-jamie 

</details>

## About the Team

<details><summary>Toggle to view information about the team.</summary>

- Team name:Condition Team
- OCTO-DE product owner:Jen Bertsch
- Product manager: Yurena Garcia
- Designer:Sudeepti Bhatnagar
- FE engineer:William Phelps
- Product/team Slack channel: https://dsva.slack.com/archives/C04AZ8T7XN1

</details>

<details><summary>Toggle to view more intake research ticket information</summary>

### Do you have an accessibility champion?

Yes, we have a champion

### If you have an accessibility champion, what's their name?

Tracy Tran

### Calendar checking

- [x] Based on this calendar, confirm we have specialists who can support your study in your desired timeframe
- [x] Confirm you are filing this support ticket at least 1 week in advance of said timeframe, so we have enough heads up to provide support

### Have you reviewed the CAIA capacity calendar?

- [x] Yes, we have reviewed the CAIA capacity calendar and can confirm specialists are available to support in our research timeframe

### When is the timeframe for your testing?

December 2nd - 13th

### Are you interested in doing a pilot of your study with a CAIA accessibility specialist before the study begins?

- [x] Yes, we are interested in doing a pilot

</details>

## Research Sessions

> [!TIP]
> - Providing CAIA A11ys with the time slots in advance for AT users, will allow a11ys to avoid conflicts on the CAIA A11y Capacity Calendar, and reach out to other a11ys, should CAIA not be available. 
> - Providing CAIA A11ys with the technology and devices the users will be testing with, will allow CAIA to prepare in advance for the sessions. 

<details><summary>Toggle to view research timing information.</summary>

### Timeframe
- **Research Timeframe**: 12/2/2024 - 12/13/2024
- [x] Pilot Requested
    - **Pilot Session Mobile**: 11/25/2024  #97582

### Length of sessions

- **Session length:** `pending` minutes
- **Buffer time between sessions:** `pending` minutes
- **Maximum sessions per day:** `pending`

</details>

###  Testing Schedule

CAIA A11Ys will be looking for device and AT information. We realize that some of this information will not be available until the day of the session. 

Please toggle and use the following table to enter the following information for each participant: Date and Time, Assistive tech (AT), Device and Operating System, Browser, and Skill level:

> [!TIP]
> 🗓 **Scheduling Source of Truth** 
> - [Schedule Available via Canvas in Slack](https://dsva.slack.com/canvas/C082SKRHAJV) 
> - **Slack Channel:** [Benefits Content Classification](https://dsva.slack.com/archives/C04AZ8T7XN1)
> - **NOTE**: You will need to be added to the channel to gain access. 

<details><summary>Toggle to view the AT testing schedule</summary>

P# | Date and Time (ET) | AT | Device and OS | Browser | Skill | CAIA A11ys Available
------------------|------------------|--------------|--------------|--------------|--------------|--------------
🧑‍✈ PILOT - MOBILE ✅  | 11/25/2024 | VoiceOver | Mobile iPhone | Safari | N/A | @mutiaadhocteam @sara-amanda 
P2 | 12/4/2024, 11 a.m. - 12:30 p.m. ET | JAWS | Desktop; Windows | Edge | Advanced | @coforma-jamie 
P3 | ~~12/4/2024, 3-4 p.m. ET~~ `no show` | Voice Control | Desktop; MacOS | Safari, Chrome, etc. `TBD` | Novice, Intermediate, Advanced `TBD` |  @coforma-jamie 
P4 | 12/5/24, 11:00 a.m. -12:30 p.m. ET | JAWS  or VoiceOver? | Desktop; Windows | Safari, Chrome, etc. `TBD` | Novice, Intermediate, Advanced `TBD` | @EvanAtCoforma 
P5 | ~~12/5/2024, 3-4 p.m. ET~~ `no show` | Alternative navigation one-handed keyboard | Mobile or Desktop; Windows, MacOS, iOS, Android, etc. `TBD` | Safari, Chrome, etc. `TBD` | Novice, Intermediate, Advanced `TBD` | @sara-amanda 
P6 | ~~12/6/2024, 11 a.m. - 12:30 p.m.~~ `no show` | JAWS | Desktop; Windows | Safari, Chrome, etc. `TBD` | Novice, Intermediate, Advanced `TBD` | @SarahKay8 
P7 | 12/6/24, 3-4 p.m. | Alternative Navigation Voice Control  | Mobile or Desktop; Windows, MacOS, iOS, Android, etc. `TBD` | Safari, Chrome, etc. `TBD` | Novice, Intermediate, Advanced `TBD` | `TBD- Based on a11y availability`
P9 `*` | 12/9/24, 3-4  p.m. ET | `Screen Magnification` | Mobile or Desktop; Windows, MacOS, iOS, Android, etc. `TBD` | Safari, Chrome, etc. `TBD` | Novice, Intermediate, Advanced `TBD` | `TBD- Based on a11y availability`
P10 | 12/10/24, 11 a.m .- 12:30 p.m.| VoiceOver | Mobile iOS | Safari, Chrome, etc. `TBD` | Novice, Intermediate, Advanced `TBD` | `TBD- Based on a11y availability`
P11 `*` | 12/10/24, 3-4 p.m. ET | `Screen Magnification` | Mobile or Desktop; Windows, MacOS, iOS, Android, etc. `TBD` | Safari, Chrome, etc. `TBD` | Novice, Intermediate, Advanced `TBD` | `TBD- Based on a11y availability`
P12 | 12/11/24, 11 a.m. - 12 p.m. ET | Alternative Navigation One-Handed Keyboard | Mobile or Desktop; Windows, MacOS, iOS, Android, etc. `TBD` | Safari, Chrome, etc. `TBD` | Novice, Intermediate, Advanced `TBD` | @coforma-jamie 

- **Consideration**: You can also consider asking participants more specific AT and device combinations. 
- **For instance:** TalkBack on Samsung Galaxy 8, or VoiceOver on iPad with magnification.

</details>

CAIA A11ys will review the coverage needed, and self-assign, based on a11y availability.

## Supporting artifacts 

<details><summary>Toggle to view the supporting artifacts</summary>

- [Product outline](https://docs.google.com/document/d/1rpP3nKa4jGE_n3UVEkS9SXf2zEYFspNBzyAuMBfterM/edit?tab=t.0#heading=h.pb8c9u795h8v)
- [User flows](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1695823278972/901fc8ca9be5136a8b4a3dabc01e0f7238cb7d83)
- [Prototype or mockups](https://www.figma.com/design/P6IUm8gN00H3Yi1XzN80fj/(526ez)-Condition-information-entry?node-id=324-9662&node-type=canvas&t=GDUqm69UjZoNKgNM-0)
- [Research plan](https://docs.google.com/document/d/1I4YAFI9xYL6EnMV6DQ5iqdEUwgCKR_TWXich9JvW-Zo/edit?tab=t.0)
- [Research conversation guide](https://docs.google.com/document/d/12bLzBBviYDFCwGKvPUU8CmESkpqbV3MrDcyik-WumeI/edit?tab=t.0#heading=h.mtc4swvlio8d)
- [Epics/issue that may be helpful](Insert URL here between the parenthesis)

</details>

## Next Steps: Definition of Done

```[tasklist]
### Initial next steps (CAIA Research Ticket Refinement)
- [x] This ticket has been received.
- [X] This ticket will be refined by CAIA Accessibility.
- [x] [Slack messaging with Team - Submission](https://dsva.slack.com/archives/C8E985R32/p1730994059147409)
- [x] [Slack messaging with Team - Scheduling](https://dsva.slack.com/archives/C8E985R32/p1731438099887199)
```

```[tasklist]
### CAIA A11Y Tasks
- [x] **Review Convo Guide**
- [x] **PILOT SESSION MOBILE: SCHEDULE** Pilot session with team.
- [x] **PILOT SESSION MOBILE: CREATE TICKET** Link ticket in this task list.
- [ ] #97582
- [x] **Generic Calendar Holds**
- [x] **CALENDAR HOLDS** Update A11y [CAIA Capacity Calendar](https://calendar.google.com/calendar/u/0/embed?src=c_f982350fc73a8162a1088ab14d00dd37335b2b559a60d6b67eb8021416f81c2a@group.calendar.google.com&ctz=America/Chicago) with holds for user research sessions with AT, when dates become available.
- [x] Update research tracker
- [x] **REMOVE CALENDAR HOLDS** Remove holds on A11y [**CAIA Capacity Calendar**](https://calendar.google.com/calendar/u/0/embed?src=c_f982350fc73a8162a1088ab14d00dd37335b2b559a60d6b67eb8021416f81c2a@group.calendar.google.com&ctz=America/Chicago) and replace with invitations from project team (or Perigean)
- [x] **Divvy user research session support with CAIA A11ys** - based on availability and technology available to support.
- [x] [**Schedule Source of Truth:** NOW AVAILABLE via Canvas in Slack](https://dsva.slack.com/canvas/C082SKRHAJV)
- [x] **Review research folder and assets**
- [x] **Participate in research** **Date Span:**`12/4/2024` and `12/13/2024`
- [x] **SESSION NOTES**: Create [CAIA notes document](https://docs.google.com/document/d/1jueOY_5qTUIqHo7lzfClrEyW11HzwHQreg9NurNWPUU/edit?usp=sharing) for a11ys to use during the sessions
- [x] **SYNC:** Connect with CAIA A11ys on research findings (meeting)
- [x] **Create/update the CAIA A11y Research Deliverable** This is dependent on the number of study participants CAIA supports. If minimal, A11Ys may share feedback only in the feedback Mural or this ticket.
- [x] **Share the completed deliverable** with the project team via Slack: `Primary Team Slack` and `Ticket`
```

```[tasklist]
### VFS TEAM Tasks
- [x] Tag @Naomi Eke in Slack [#sitewide-content-accessibility-ia](https://dsva.slack.com/archives/C01K37HRUAH)
- [x] **Request** a mobile pilot session with CAIA a11ys
- [x] **Request** feedback on convo guide
- [x] **Request** feedback on research guide
- [x] **Schedule** a mobile pilot session with CAIA a11ys
- [x] Receive feedback on Research Plan for Screen Readers (if requested)
- [x] Receive feedback on Conversation Guide for Screen Readers (if requested)
- [x] **PROTOTYPE:** Provide prototype to CAIA A11ys via this ticket in the supporting artifacts section - `Staging`
- [x] **TESTING DATES & TIMES NEEDED** See the table in the ticket above. Please provide date range and slots for AT sessions being planned.
- [x] Provide the [**Schedule Source of Truth:** Available via Canvas in Slack](https://dsva.slack.com/canvas/C082SKRHAJV)
- [x] **INVITES:** Forward invites from Perigean to CAIA A11ys (Sara)
- [x] **ASSISTIVE TECH. INFO NEEDED:** Please provide the AT the users will be utilizing during the research sessions in the table in the ticket above.
```

**Comment by sudeeptibh** [comment]
Also posted in slack, but checking if 11/25 at 9AM PST works for the pilot with Maria. Thanks!

**Comment by sudeeptibh** [comment]
Sign up sheet for sessions can be found on this [Canvas](https://dsva.slack.com/canvas/C082SKRHAJV). Thank you for your support!

**Comment by sara-amanda** [comment]
## Session Support

> Sign up sheet for sessions can be found on this [Canvas](https://dsva.slack.com/canvas/C082SKRHAJV). Thank you for your support!

Thank you, @sudeeptibh! We are working to fill those sessions for you. 

### This Week

This week we can cover 5 out of the 6 available sessions. We are working to see if we can get that last one filled.  

### Next Week

Next week, so far we have 1 session covered out of the 4 sessions. Two are magnifiers, which don't tend to need as much support, but for the remaining sessions. If we can get the last one covered, we will be sure to update your canvas in Slack. 

🙌 _Thank you for working with us! We look forward to supporting your study!_ 

All the best!

Sara

cc: @SarahKay8 @coforma-jamie @EvanAtCoforma @rileyorr @mutiaadhocteam

**Comment by coforma-jamie** [comment]
This study is complete. Ready for closure validation @sara-amanda

**Comment by sara-amanda** [comment]
> [!NOTE]
> CAIA INTERNAL
> - [x] This ticket has been added to the intake's sub-issue section for connectivity
> - [x] This work has been completed
> - [x] Closing ticket
> - [x] Adding to tracker for CAIA Sprint 140

**Comment by sudeeptibh** [comment]
Thank you all for your help!
